src/FirstSteps/RPG/Adventures/Hell.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FirstSteps.RPG.Heroes;
+using FirstSteps.RPG.Tools;
+using Spectre.Console;
+using System;
+
+namespace FirstSteps.RPG.Adventures
+{
+    public class Hell
+    {
+        public void FightWithboss(Hero hero)
+        {
+            Boss boss = new Boss("Diablo")
+            {
+                Health = 55,
+                Damage = 6,
+                Strength = 4,
+            };
+            Display.DefaultText($"You entered to the Hell!");
+
+            while (boss.Health >= 0)
+            {
+                string userInput = AnsiConsole.Prompt(
+                    new SelectionPrompt<string>()
+                      .Title($" \nChose what u want to do! {Emoji.Known.CrossedSwords}")
+                      .AddChoices("Boss stats", "Hit the boss!", "Run"));
```

Comment:
You can extract commands to the const and use it in all places. In the future when you will want to change it you will update only one place.